waku/node/v2/waku_types.nim

```diff
@@ -16,7 +16,30 @@ type
   # NOTE based on Eth2Node in NBC eth2_network.nim
   WakuNode* = ref object of RootObj
     switch*: Switch
-    # XXX Unclear if we need this
     peerInfo*: PeerInfo
     libp2pTransportLoops*: seq[Future[void]]
-    messages*: seq[(Topic, Message)]
+  # TODO Revist messages field indexing as well as if this should be Message or WakuMessage
+    messages*: seq[(Topic, WakuMessage)]
+
+  WakuMessage* = object
+    payload*: seq[byte]
+    contentTopic*: string
+
+
+
```

Comment:
Seems like an excess of new lines

Reply:
Deliberate to space out different parts of the code, like sections. Makes it easier to visually scan IMO. Also makes code folding look better.